src/Symfony/Component/Yaml/Parser.php

```diff
@@ -1083,4 +1083,17 @@ private function getLineTag($value, $flags, $nextLineCheck = true)
 
         throw new ParseException(sprintf('Tags support is not enabled. You must use the flag `Yaml::PARSE_CUSTOM_TAGS` to use "%s".', $matches['tag']), $this->getRealCurrentLineNb() + 1, $value, $this->filename);
     }
+
+    private function getDeprecationMessage($message)
+    {
+        $message = rtrim($message, '.');
+
+        if (null !== $this->filename) {
+            $message .= ' in '.$this->filename;
+        }
+
+        $message .= ' on line '.($this->getRealCurrentLineNb() + 1);
```

Comment:
Not sure if it makes sense to specify the line if we don't know the filename.

Reply:
I guess it can be helpful if someone parses a big YAML file, but reads the file contents themselves (thus using `Yaml::parse()` instead of `Yaml::parseFile()`).

Reply:
we reverted that before because it cluttered deprecation logs, so same pov as fab here to me

Reply:
At the time we did that we didn't have the filenames. Now we have them because internally we use `parseFile()` instead of `parse()`. So having line numbers without filenames just affects users using the YAML parser in their own code (and there they probably know the filenames).

Reply:
And by the way, when using the lint command you will get the line numbers too now (you wouldn't before).

Reply:
ok, fair enough